src/sos/docker/client.py

```diff
@@ -48,117 +49,6 @@ def total_memory(self, image='ubuntu'):
             self.tot_mem = None
         return self.tot_mem
 
-    def _is_image_avail(self, image):
```

Comment:
I do not know what you are doing here. You move the function to a separate module, but I do not see the related changes when this function is called.